pkg/api/aim2/api/request/run.go

```diff
@@ -49,20 +49,14 @@ type SearchRunsRequest struct {
 	ExcludeTraces bool   `query:"exclude_traces"`
 }
 
-// MetricTuple represents a metric with key and context.
-type MetricTuple struct {
-	Key     string `query:"metric"`
-	Context string `query:"context"`
-}
-
 // SearchMetricsRequest is a request struct for `GET /runs/search/metric` endpoint.
 type SearchMetricsRequest struct {
 	BaseSearchRequest
-	Metrics    []MetricTuple `query:"m"`
-	Query      string        `query:"q"`
-	Steps      int           `query:"p"`
-	XAxis      string        `query:"x_axis"`
-	SkipSystem bool          `query:"skip_system"`
+	Metrics    [][]string `json:"metrics"`
```

Comment:
Possibly we should keep `MetricTuple` as a synonym for `[]string`, so this struct field becomes `[]MetricTuple`?